cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/AwsClusterSettingsStep.tsx

```diff
@@ -56,21 +56,16 @@ const nodeInstanceTypes: NodeInstanceType[] = [
 interface AwsClusterSettingFormInputs {
     [AWS_FIELDS.CLUSTER_NAME]: string;
     [AWS_FIELDS.NODE_PROFILE]: string;
-    [AWS_FIELDS.IMAGE_INFO]: string;
+    [AWS_FIELDS.OS_IMAGE]: string;
 }
 
 function createYupSchemaObject() {
     return {
-        IMAGE_INFO: yupStringRequired('Please select an OS image'),
```

Comment:
I wonder if this is the place to make OS_IMAGE required?

Reply:
Well, I do have a question about the validation part. As we are gonna set default value to those input fields.(os images and node profile). Is that necessary to have validation here?